presto-docs/src/main/sphinx/release/release-344.md

```diff
@@ -0,0 +1,64 @@
+# Release 344 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`murmur3` function. ({issue}`5054`)
+* Add {func}`from_unixtime_nanos` function. ({issue}`5046`)
+* Add [T-Digest](/functions/tdigest) type and functions. ({issue}`5158`)
+* Improve performance and latency of queries leveraging dynamic filters. ({issue}`5081`)
+* Allow to configure (using ``dynamic-filtering.service-thread-count`` feature toggle) number of
+  threads used for processing dynamic filters on coordinator. ({issue}`5341`)
+* Extend environment variables substitution (`${ENV:variable-name}`) to work also when the substituted text
+  is part of the configuration value. ({issue}`4345`)  
+* Remove `fast-inequality-joins` configuration property. The fast inequality joins remain enabled. ({issue}`5375`)
+* Report dynamic filters collection duration. ({issue}`5434`)  
+* Improve query performance by propagating more detailed dynamic filters to coordinator. ({issue}`5340`)  
+* Improve performance and accuracy of {func}`approx_percentile`. ({issue}`5158`)
+* Improve performance of cross join queries. ({issue}`5276`)
+* Prevent potential query deadlock when query runs out of memory.  ({issue}`5289`)
```

Comment:
How does it manifest? How do i recognize situations where this would help me, so that I feel even more desire for an upgrade?

Reply:
This might happen `PartitionedOutputOperator` fails to allocate more memory. I'm not sure what more (in user understandable way) can be added here.